只在 cmmusic-1.6-ylmf1.new 存在：config.h
只在 cmmusic-1.6-ylmf1.new 存在：config.log
只在 cmmusic-1.6-ylmf1.new 存在：config.status
只在 cmmusic-1.6-ylmf1.new 存在：Makefile
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic-ConfigMgr.o
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic-FileMgr.o
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic-Helper.o
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic-LrcMgr.o
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic-main.o
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic-PlayerMgr.o
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic-PluginMgr.o
只在 cmmusic-1.6-ylmf1.new/src 存在：cmmusic-UIMgr.o
diff -ur cmmusic-1.6-ylmf1/src/commdef.h cmmusic-1.6-ylmf1.new/src/commdef.h
--- cmmusic-1.6-ylmf1/src/commdef.h	2009-06-21 20:35:14.000000000 +0800
+++ cmmusic-1.6-ylmf1.new/src/commdef.h	2011-11-09 16:11:57.932652584 +0800
@@ -34,14 +34,14 @@
 //uimgr.h
 
 #ifdef HAVE_LIBNCURSESW
-#include <ncursesw/curses.h>
-#include <ncursesw/panel.h>
-#include <ncursesw/menu.h>
+#include <curses.h>
+#include <panel.h>
+#include <menu.h>
 #else
 #ifdef HAVE_LIBNCURSES
-#include <ncurses/curses.h>
-#include <ncurses/panel.h>
-#include <ncurses/menu.h>
+#include <curses.h>
+#include <panel.h>
+#include <menu.h>
 #else
 #include <curses.h>
 #include <panel.h>
只在 cmmusic-1.6-ylmf1.new/src 存在：.deps
只在 cmmusic-1.6-ylmf1.new/src 存在：Makefile
只在 cmmusic-1.6-ylmf1.new 存在：stamp-h1
